ansible/playbooks/aix.yml

```diff
@@ -256,21 +264,32 @@
           register: java8
           tags: java8
 
-        - name: Create /usr/java8_64 if it does not exist
-          file:
-            path: /usr/java8_64
-            state: directory
-          when: java8.stat.isdir is not defined
-          tags: java8
-
         - name: Transfer and Extract AdoptOpenJDK 8
           unarchive:
             src: https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl={{ bootjdk }}&os=aix&arch=ppc64&release=latest&type=jdk
-            dest: /usr/java8_64
+            dest: /usr
             remote_src: yes
           when: java8.stat.isdir is not defined
           tags: java8
 
+        - name: Find java8 directory
+          find:
+            file_type: directory
+            paths: /usr
+            patterns: 'jdk8u*'
+          register: java8_files_matched
+          tags: java8
+
+        - name: Rename to java8_64
+          command: mv "{{ item.path }}" /usr/java8_64
+          with_items:
+            - "{{ java8_files_matched.files }}"
+          tags: java8
+
+        - name: Setting Java 8 as default
+          command: ln -s /usr/java8_64/jre/bin/java usr/bin/java
```

Comment:
Not to my knowledge ref `update-alternatives`. Are you sure any system updates would overwrite a link that's already there? We're doing this because we're not installing any "system" java 8

Reply:
Unless there's evidence to suggest that there is a setup which will prevent this working as is, I'm disinclined to change it, or the system PATH settings on the machine. I don't think it's unreasonable to have `/usr/bin/java` pointing at a "reasonable" (i.e. supported) implementation of java in the default path. If an update replaces it with something that works, then I'm ok with that (although I doubt an update would suddenly install a java when one wasn't there previously), but I want to ensure we're covered if there isn't anything there.

Reply:
If you're happy to deal with any fall out of making a change not to put the symlink in place in /usr/bin (we previously installed IBM java into /usr/java8_64 and symlinked `/usr/bin/java` to that) then I'm happy to take a subsequent PR but at the moment since since I'm single-handedly dealing with all build and infra issues at adopt I'm approving this on the basis it's least likely to cause me any follow-on problems.